ee/clickhouse/queries/diagnose/diagnose.py

```diff
@@ -3,6 +3,14 @@
 from posthog.queries.diagnose import DiagnoseResponse
 
 
+def get_people_by_success_by_event_visited(team_id: int, target_entity: str, source_entity: str):
+    return [{"event": "signup", "success": 123, "failure": 456}]
+
+
+def get_entity_odds_ratio(event_stats):
+    return 1
+
+
 class ClickhouseDiagnose:
     _filter: DiagnoseFilter
```

Comment:
We're trying to move towards unifying filters. Specially, since this filter (limited to funnels) uses the same filter as funnels, makes sense to use `Filter` here as well? We'll add the 2 new properties to `Filter` itself